app/config.go

```diff
@@ -19,10 +19,12 @@ func SetupConfig(config interface{}) {
 		files = []struct {
 			Path      string
 			Unmarshal file.Unmarshal
+			Optional  bool
 		}{
 			{
 				Path:      ConfigFilename,
 				Unmarshal: json.Unmarshal,
+				Optional:  false,
```

Comment:
shouldn't this be true to keep retro-compatibility?

Reply:
Yes, it should. Well spotted

Reply:
This field was added in version v0.4.0 and since we never used it, I assumed that `false` (zero-value) would be the correct solution. But `true` is a safer Option, pun intended.